requirements.txt

```diff
@@ -14,4 +14,5 @@ dlib>=19.20.0
 retina-face>=0.0.1
 mediapipe>=0.8.7.3
 fire>=0.4.0
-gunicorn>=20.1.0
\ No newline at end of file
+gunicorn>=20.1.0
+pyheif>=0.7.1

```

Comment:
requirements.txt is just showing dependencies. we are not using this in the installation of the library. this dependency should be added in setup.py.